BOC_schema.sql

```diff
@@ -0,0 +1,218 @@
+-- ---
+-- Globals
+-- ---
+
+-- SET SQL_MODE="NO_AUTO_VALUE_ON_ZERO";
+-- SET FOREIGN_KEY_CHECKS=0;
+
+-- ---
+-- Table 'BOC_User-Session-jt'
+-- 
+-- ---
+
+DROP TABLE IF EXISTS "BOC_User-Session-jt" CASCADE;
+		
+CREATE TABLE "BOC_User-Session-jt" (
+  id SERIAL,
+  session_id INTEGER,
+  user_id INTEGER,
+  user_done_ordering BOOLEAN,
+  PRIMARY KEY (id)
+);
+
+-- ---
+-- Table 'BOC_Users'
+-- 
+-- ---
+
+DROP TABLE IF EXISTS "BOC_Users" CASCADE;
+		
+CREATE TABLE "BOC_Users" (
+  user_id SERIAL,
+  first_name VARCHAR(90),
+  last_name VARCHAR(90),
+  email VARCHAR(90),
+  password VARCHAR(25),
+  PRIMARY KEY (user_id)
+);
+
+-- ---
+-- Table 'BOC_Sessions'
+-- 
+-- ---
+
+DROP TABLE IF EXISTS "BOC_Sessions" CASCADE;
+		
+CREATE TABLE "BOC_Sessions" (
+  session_id SERIAL,
+  host_id INTEGER,
+  rollover_session_id INTEGER,
```

Comment:
what is difference between session_id and rollover_session_id?

Reply:
I think rollover_session_id is a stretch goal, so nothing important atm